src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationRefresherProvider.cs

```diff
@@ -26,20 +39,24 @@ public AzureAppConfigurationRefresherProvider(IConfiguration configuration, ILog
                         // Use _loggerFactory only if LoggerFactory hasn't been set in AzureAppConfigurationOptions
                         if (refresher.LoggerFactory == null)
                         {
-                            refresher.LoggerFactory = _loggerFactory;
+                            refresher.LoggerFactory = loggerFactory;
                         }
 
                         refreshers.Add(refresher);
                     }
-                }
-            }
 
-            if (!refreshers.Any())
-            {
-                throw new InvalidOperationException("Unable to access the Azure App Configuration provider. Please ensure that it has been configured correctly.");
-            }
+                    PropertyInfo p = typeof(ChainedConfigurationProvider).GetProperty("Configuration", BindingFlags.Public | BindingFlags.Instance);
```

Comment:
We need to do this only if the provider is ChainedConfigProvider. I think it can be placed inside an `else if` block:
```
if (provider is IConfigurationRefresher refresher)
{

}
else if (provider is ChainedConfigurationProvider chainedProvider)
{
  // Reflection
}
```